pyproject.toml

```diff
@@ -1,2 +1,88 @@
 [tool.black]
 line-length = 100
+
+[tool.ruff]
+# 3.10 introduces standards such as PEP604 which changes lots of typing
+target-version = "py39"
+line-length = 100
+select = [
+  "ANN",
+  "ARG",
+  "B",
+  "C",
+  "D",
+  "DTZ",
+  "E",
+  "F",
+  "I",
+  "ICN",
+  "ISC",
+  "N",
+  "PLC",
+  "PLE",
+  "PLR",
+  "PLW",
+  "Q",
+  "RUF",
+  "S",
+  "T",
+  "TID",
+  "UP",
+  "W",
+  "YTT",
+]
+unfixable = [
+  # Don't touch unused imports
+  "F401",
+]
```

Comment:
I don't fully understand this field even after checking the docs. Will this only ignore unused imports that we noqa? (vs other noqas may show up as warnings somewhere?)

Reply:
These rules are not acted on when running `ruff check --fix`. The reason this was added is that it causes a lot of headaches when developing. This will just label it as a linting issue and not touch it when auto fixing issues.

Reply:
I see. So it will still get flagged by the linter? Guess I'm confused since it seems that the _version.py file works without a noqa now